librepo/yum.c

```diff
@@ -20,32 +20,27 @@
 
 #define _POSIX_SOURCE
 #define _DEFAULT_SOURCE
+#define  BITS_IN_BYTE 8
```

Comment:
everyone knows this, why does it need to be defined!?

Reply:
Avoiding magic number.